docs/string-representation.md

```diff
@@ -559,4 +559,4 @@ namespace icu {
 
 }; // namespace icu
 
-```
\ No newline at end of file
+```
```

Comment:
Not sure what I actually changed here, I didn't mean to change this.

Reply:
Looks like you added a missing line feed to the end of the file. Thansk.